### PR TITLE
Add Dune analytics link to landing Navbar

### PR DESCRIPTION
**Pull request by Diogomartf**
image

### CODE REVIEW

**Review by ElRodrigote — APPROVED** [review]

File: packages/landing/components/navbar/MobileMenu.tsx [hunk 1]
```diff
@@ -45,6 +45,16 @@ export default function MobileMenu({
             onClick={toggle}
           ></div>
           <div className="absolute left-0 w-full gap-2 px-6 py-2 border-b border-solid rounded-lg bg-surface-25 top-nav-height border-surface-75">
+            <Link
```
[the remark this answers is not in the document]
Comment (Diogomartf) [reply packages/landing/components/navbar/MobileMenu.tsx:48]: we can add those directly on `Link` here

[the remark this answers is not in the document]
Comment (ElRodrigote) [reply packages/landing/components/navbar/MobileMenu.tsx:48]: Yeah, that too. I was thinking something like create that component to use it as a single point for all the external links we have, so all those links have that `rel` value when `target="_blank"` is used.

**Comment by berteotti** [comment]
I suggest adding a external link icon to analytics. What do you think?

File: packages/landing/components/navbar/MobileMenu.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Diogomartf) [reply packages/landing/components/navbar/MobileMenu.tsx:48]: Yeah, in this case, I don't think it's worth to create a component, if we end up creating more external links we can add them.

**Review by berteotti — APPROVED** [review]

**Pull request merged**